common.yaml

```diff
@@ -313,13 +186,8 @@ packages:
  - systemd-journal-remote
  # Extras
  - systemd-journal-gateway
- # RHEL7 compatibility
- - compat-openssl10
- # Make sure we pull in at least clevis 15; it drops the rd.neednet=1 hardcode
- # and has a few other patches we need.
- # https://bugzilla.redhat.com/show_bug.cgi?id=1853651
- - "'clevis >= 15-1.el8' 'clevis-luks >= 15-1.el8' 'clevis-dracut >= 15-1.el8'"
- - cryptsetup-reencrypt tpm2-tools
```

Comment:
This is dropping cryptsetup-reencrypt. Is that on purpose? Would be better to have a separate commit with a rationale.

Reply:
Moved to https://github.com/openshift/os/pull/850